doc/antora.yml

```diff
@@ -2,6 +2,6 @@ name: cider-nrepl
 title: CIDER nREPL
 # The version should be in the format major.minor (e.g. 0.28).
 # Don't include patch in the version (e.g. 0.28.7).
-version: "0.31"
+version: "0.37"
```

Comment:
I kept thinking about this and it seems to introduce a bunch of complexity for an automated check.

Removing master [from here](https://github.com/clojure-emacs/docs.cider.mx/blob/bf4e4bb6e811c978453837b417987ac93e0916d3/antora-playbook.yml#L14) would enable a lot of simplicity - then, the this value would only change from digit release to digit release, instead of back and forth between digits and `~`.

Let me doubt cider-nrepl master docs have value anyway - not in the same way CIDER's do. We don't publish snapshots, and the intended audience is much smaller.

Reply:
It would not, if we made https://docs.antora.org/antora/latest/playbook/urls-latest-version-segment/ work?

i.e. `/latest` would point at the latest tag. And there would be no `master` docs.

I could look into it.